specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json

```diff
@@ -534,9 +505,12 @@
           "description": "The column of the table over which the given query should run",
           "type": "string"
         },
-        "filter": {
-          "description": "A query to to run over the provided table and column to purge the corresponding data.",
```

Comment:
Changing this name is a breaking change in the libraries that are generated from the Swagger. You can see where it breaks the existing Go library: https://github.com/Azure/azure-sdk-for-go/pull/1505/files#diff-1f11d853f4bf937b5f3121aabcf3d424L737

Reply:
I should have provided some more context to this PR. The API we exposed for purging data was pulled due to a security issue. The currently documented API no longer exists. Given that's the case, this change is to update the actual offered API, which I realize is breaking unfortunately.